src/app/features/StepDefinitionEditor/Models/ComponentInstanceModel.js

```diff
@@ -46,6 +46,11 @@ var createComponentInstanceModelConstructor = function (
                 get: function () {
                     return toAST.call(this);
                 }
+            },
+            link: {
```

Comment:
same as above -> pulled this one out of this pull request.Will do more analysis and submit changes in later pull request.